src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -38,23 +41,77 @@ import scala.annotation.tailrec
 
 object Homework :
 
+  //solution for booleans
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean =
+      if (b)
+        false
+      else
+        true
+
+    def and(left: Boolean, right: Boolean): Boolean =
+      if (left)
+        if (right)
+          true
+        else
+          false
+      else
+        false
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def or(left: Boolean, right: Boolean): Boolean =
+      if (left)
+          true
+      else
+          if (right)
+            true
+          else
+            false
 
-    def or(left: Boolean, right: Boolean): Boolean = ???
 
   end `Boolean Operators`
 
+  //solution for Fermat numbers
   object `Fermat Numbers` :
 
-    val multiplication: (BigInt, BigInt) => BigInt = ???
+    val multiplication: (BigInt, BigInt) => BigInt = (a,b) =>
+    {

```

Comment:
Code format: let's keep open `{` on the same line with the previous code. Please fix it everywhere.